simple_history/models.py

```diff
@@ -421,7 +421,7 @@ def get_default_history_user(instance):
 
         extra_fields = {
             "history_id": self._get_history_id_field(),
-            "history_date": models.DateTimeField(),
+            "history_date": models.DateTimeField(db_index=True),
```

Comment:
Can we make this a setting that is False by default in django settings? And then can we add a param that allows for model level override?

Reply:
ok